src/pages/Home.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaRegArrowAltCircleRight } from 'react-icons/fa';
+import { fetchApiData } from '../redux/metrics/metricsSlice';
+import { fetchDetailData } from '../redux/details/detailsSlice';
+import '../styles/Home.css';
+
+function Home() {
+  const dispatch = useDispatch();
+  const metrics = useSelector((store) => store.metrics.apiData);
+  const loading = useSelector((store) => store.metrics.isLoading);
```

Comment:
**OPTIONAL** - To help improve the readability of your code, I suggest you get the `metrics` state from your store once and destructure all your required properties. Please fix this 🙏

Reply:
fixed